kubeflow/tf-serving/tf-serving-service-template.libsonnet

```diff
@@ -50,12 +50,12 @@
         ports: [
           {
             name: "grpc-tf-serving",
-            port: 9000,
+            port: params.portGRPC,
```

Comment:
The e2e test failed, it might be that the type should be int not string (I am not sure)

Reply:
I checked the cla is signed😅

I tried to use int instead of string but then it wouldn’t compile. Ist there another keyword for int in ks?